fili-core/src/main/java/com/yahoo/bard/webservice/web/DataApiRequest.java

```diff
@@ -114,6 +116,8 @@
 
     private final DruidFilterBuilder filterBuilder;
 
+    private Optional<OrderByColumn> dateTimeSort = Optional.empty();
```

Comment:
Nope. It can't be final. Because, its value will be changed if there is a sort request for dateTime column. After the code optimization, we don't need to initialize this.